frontends/concrete-python/concrete/fhe/compilation/utils.py

```diff
@@ -20,39 +15,6 @@
 # ruff: noqa: ERA001
 
 
-def print_ctrl_c_message():  # pragma: no cover
```

Comment:
Is thins no longer needed when not using HPX?

Reply:
Printing is now done in C++, no need for this.